homeassistant/components/cloud/alexa_config.py

```diff
@@ -204,10 +204,10 @@ def _migrate_alexa_entity_settings_v1(self) -> None:
             # Don't migrate if there's a YAML config
             return
 
-        for entity_id in {
+        for entity_id in (
```

Comment:
Why do we change this?

Reply:
https://beta.ruff.rs/docs/rules/iteration-over-set/

Reply:
Ok. I'm not sure we should make this change as the members of the container may not be the same in the tuple compared to the set. It's not just a performance optimization. I haven't looked into the details of the parts that make up the container.

Reply:
Yeah it seems like this rule should only apply to literals. It'd probably be reasonable to open an issue in Ruff.